submissions/Zakkarat/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,78 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+  name: "Paolo",
+  species: "dog",
+  gender: 'male',
```

Comment:
Try define constants for repeated properties. But in general:
```
const INHABITANT = {
 species: {
  human: 'human'
 ... and so one
 }
}
```

Reply:
Sorry, could not understand this part. Could you clarify a bit?

Reply:
@Zakkarat I mean define object with constants, then use it for inhabitants.
```javascript
const INHABITANT = {
    species: {
        dog: 'dog',
        human: 'human',
    },
    genders: {
        male: 'male',
        female: 'female',
    }
};
const dog = {
    name: "Paolo",
    species: INHABITANT.species.dog,
    gender: INHABITANT.genders.male,
    legs: 4,
    hands: 0,
    saying: "Bark",
};
```

Reply:
Thank you! Done!